src/nocode/main.cpp

```diff
@@ -120,14 +123,31 @@ void printSymbols(nc::core::Context &context, QTextStream &out) {
     }
 }
 
-void printRegionGraphs(nc::core::Context &context, QTextStream &out) {
+void printRegionGraphs(nc::core::Context& context, QTextStream &out) {
     out << "digraph Functions { compound=true; " << endl;
     foreach (const auto *function, context.functions()->list()) {
         context.graphs()->at(function)->print(out);
     }
     out << "}" << endl;
 }
 
+void printRanges(nc::core::Context& context, QTextStream &out) {
+    auto document = new nc::core::likec::CxxDocument(&context);
+
+    foreach (const auto &instr, context.instructions()->all()) {
+        auto addr = QString("%1").arg(instr->addr(), sizeof(unsigned long long int), 16, QChar('0'));
```

Comment:
And why sizeof(unsigned long long it)?
It is field width, so, it must be in hex characters.
One hex character represents four bits.
So, I guess you meant `sizeof(instr->addr()) * CHAR_BIT / 4`?